bylaws/002-psr-workflow.md

```diff
@@ -48,6 +48,10 @@ Once four weeks have passed and two viable trial implementations can be demonstr
 
 If the Acceptance Vote passes, then the proposal officially becomes an accepted PSR. At this time the Working Group is automatically dissolved, however the Editor's input (or a nominated individual communicated to the secretaries) may be called upon in the future should typos, changes or Errata on the specification be raised.
 
+## Erratas
+
+If an incongruence or the need for clarifications arise after the PSR acceptance, it is still possible to amend the document with an errata. The editor of the PSR has the power to handle those situations, but any meaningful change has to go through an Errata vote before merging. Typos and any other edits that do not change the meaning of the text do not fall in this category, and can be merged by the Secretaries at will.
+
```

Comment:
```suggestion

Errata clarifications may only be added to the meta document, not to the spec itself, as items in their own section.  The spec itself may only be minimally edited to point readers toward a relevant errata if appropriate.

```

Reply:
(The whitespace on this may be screwy, GitHub's editor doesn't make it easy to tell if I got the newlines right; feel free to correct that.)